README.adoc

```diff
@@ -146,6 +146,8 @@ include::finish/src/main/liberty/config/userRegistry.xml[tags=**]
 
 The registry has four users, `bob`, `alice`, `carl` and `dave`, and four groups, `Employee`, `TeamLead`, `Manager` and `PartTime`. Each user belongs to one or more groups.
 
+It is not recommended to store passwords in plain text. For simplicity reasons, the application in this guide uses XOR encoding to encode the passwords, as you can see in the `userRegistry.xml`. The passwords are encoded using the Liberty `securityUtility` command.
```

Comment:
Suggested update: It is not recommended to store passwords in plain text. The application in this guide uses XOR encoding to encode the passwords. To encode passwords, use the Liberty `securityUtility` command.

Reply:
@yeekangc, @justineechen, comment added.

Reply:
Thank you!

Reply:
"It is not recommended to store passwords in plain text. The passwords in the `userRegistry.xml` file are encoded using the Liberty securityUtility command with XOR encoding."

What about the above?

Reply:
@yeekangc, your suggestion looks good to me.